app/services/candidate_api/serializers/v1_2.rb

```diff
@@ -32,14 +32,25 @@ def serialize(candidates)
       end
 
       def query
+        subquery = Candidate
```

Comment:
1. woahhh!
2. suggest extracting to a method with a name indicating what the subquery represents

Reply:
Done.